src/core/component.js

```diff
@@ -31,11 +31,20 @@ var upperCaseRegExp = new RegExp('[A-Z]+');
  *         mapped attribute of the component plus applying defaults and mixins.
  */
 var Component = module.exports.Component = function (el, attrValue, id) {
+  var self = this;
   this.el = el;
   this.id = id;
   this.attrName = this.name + (id ? '__' + id : '');
   this.el.components[this.attrName] = this;
   this.updateProperties(attrValue);
+  this.throttledEmitComponentChanged = utils.throttle(function (oldData) {
+    el.emit('componentchanged', {
+      id: self.id,
+      name: self.name,
+      newData: self.getData(),
+      oldData: oldData
+    }, true);
+  }, 200);
```

Comment:
200 is arbirtrary yes. 5 times per second is actually plenty. I was tempted to limit at 2 per second. If you really want to have finer control I would just call `getAttribute` on a `tick` method instead of relying on th `componentchanged` event